src/OpenOfficeConvert/OOStyles.cs

```diff
@@ -2123,14 +2123,7 @@ private void CreateRightGuideword(int i)
 
                 _writer.WriteAttributeString("draw:z-index", "1");
                 _writer.WriteStartElement("draw:text-box");
-                if (_projInfo.ProjectInputType.ToLower() == "dictionary")
-                {
-                    _writer.WriteAttributeString("fo:min-height", "19.00pt"); //added for TD-4006 14.14
-                }
-                else
-                {
-                    _writer.WriteAttributeString("fo:min-height", "14.14pt"); //added for TD-2579
-                }
+	            _writer.WriteAttributeString("fo:min-height", "19.00pt");

```

Comment:
What does setting the min-height do? Should we comment this?

Reply:
No, We shouldn't comment this min-height. The default size is needed for frame on Header